src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/config/SecurityConfiguration.kt

```diff
@@ -117,10 +116,12 @@ class SecurityConfiguration<S : Session> : WebSecurityConfigurerAdapter() {
 
       request.session.invalidate()
 
-      response.sendRedirect(authLogoutSuccessUri)
+      response.sendRedirect(ssologoutUri())
     }
   }
 
+  private fun ssologoutUri() = authLogoutSuccessUri.plus("/sign-out")
```

Comment:
<pedantic>I think this should be `ssoLogoutUri`</pedantic>

Reply:
Yes it should. Ta.  There is another tweak I would like to make so will sort out then.